### ISSUE TITLE
Moving windows between columns/rows?

### ISSUE THREAD

**Issue by 0p3r4t0r** [qtile/qtile#1434]
Is there a way to move windows between columns/rows on a single screen?

**Comment by whatacold** [comment]
Yes, I've defined below hotkeys to move around:

    keys = [
        # Switch between windows in current stack pane(only pointer is moved around)
        Key([mod], "h", lazy.layout.left()),
        Key([mod], "j", lazy.layout.down()),
        Key([mod], "k", lazy.layout.up()),
        Key([mod], "l", lazy.layout.right()),
    ]

But I'm a bit confused by what it counts as `up/down`, because when I hit `Mod+h/l`,
it switches between the window `b` and `c` with below layout:

    --------
        | b |
      a |---|
        | c |
    ---------

**Comment by 0p3r4t0r** [comment]
I've tried adding those lines to my config before I made this post, but they don't move any of the windows for me. My initial setup is like this ...

```
-----------
  a   |  b
 (c)  |  
      |
------------
```

I have 3 windows with window a on top and window c below it.
I want to move window a to column 2 to get the setup below.

```
-----------
  c   |  a
      |  (b)
      |
------------
```

**Comment by pawamoy** [comment]
Try the `shuffle_right` command (instead of just `right`)?

**Comment by pawamoy** [comment]
This issue could be closed because we already have #1360. Meanwhile @0p3r4t0r, you can find the available commands for your layout at http://docs.qtile.org/en/latest/manual/ref/layouts.html, by clicking on `[source]` for a given layout and looking for methods starting with `cmd_`.
